### ISSUE TITLE
Feature Request: VS Code Extension

### ISSUE THREAD

**Issue by tsheaff** [lesshint/lesshint#472]
**What did you expect to happen?**
To find an extension in the VS Code extension search

**What's the actual result?**
![screen shot 2018-05-06 at 11 37 32 pm](https://user-images.githubusercontent.com/1238684/39688206-79ce675c-5186-11e8-9a24-5ba411f975a3.png)

**Would you be interested in submitting a PR for this issue?**
I would be down to try, but I've never written an IDE extension before. Anyone have experience with it?

**Comment by tsheaff** [comment]
By the way, the default `LESS` package in VS Code contains the following options:

```
  // Invalid number of parameters
  "less.lint.argumentsInColorFunction": "error",

  // Do not use width or height when using padding or border
  "less.lint.boxModel": "ignore",

  // When using a vendor-specific prefix make sure to also include all other vendor-specific properties
  "less.lint.compatibleVendorPrefixes": "ignore",

  // Do not use duplicate style definitions
  "less.lint.duplicateProperties": "ignore",

  // Do not use empty rulesets
  "less.lint.emptyRules": "warning",

  // Avoid using 'float'. Floats lead to fragile CSS that is easy to break if one aspect of the layout changes.
  "less.lint.float": "ignore",

  // @font-face rule must define 'src' and 'font-family' properties
  "less.lint.fontFaceProperties": "warning",

  // Hex colors must consist of three or six hex numbers
  "less.lint.hexColorLength": "error",

  // Selectors should not contain IDs because these rules are too tightly coupled with the HTML.
  "less.lint.idSelector": "ignore",

  // IE hacks are only necessary when supporting IE7 and older
  "less.lint.ieHack": "ignore",

  // Avoid using !important. It is an indication that the specificity of the entire CSS has gotten out of control and needs to be refactored.
  "less.lint.important": "ignore",

  // Import statements do not load in parallel
  "less.lint.importStatement": "ignore",

  // Property is ignored due to the display. E.g. with 'display: inline', the width, height, margin-top, margin-bottom, and float properties have no effect
  "less.lint.propertyIgnoredDueToDisplay": "warning",

  // The universal selector (*) is known to be slow
  "less.lint.universalSelector": "ignore",

  // Unknown property.
  "less.lint.unknownProperties": "warning",

  // Unknown vendor specific property.
  "less.lint.unknownVendorSpecificProperties": "ignore",

  // When using a vendor-specific prefix also include the standard property
  "less.lint.vendorPrefix": "warning",

  // No unit for zero needed
  "less.lint.zeroUnits": "ignore",

  // Enables or disables all validations
  "less.validate": true
```

But there's not thing for a lot of the nice features of `lesshint`

**Comment by jwilsson** [comment]
> I would be down to try, but I've never written an IDE extension before. Anyone have experience with it?

That sounds like a neat idea! I'm afraid I don't have any experience with creating VS Code extensions either but looking at [vscode-stylelint](https://github.com/shinnn/vscode-stylelint), it seems to be pretty straight forward.

If you decide to create it, please send us a PR with a link to it and we'll include it in our "Complementary tools" section.

> Would also love a fix-on-save option like there is for eslint or other linter plugins 😍

Also a nice idea, I'll create a new issue for tracking this.

**Comment by gnijuohz** [comment]
Found one here: https://github.com/tohjustin/vscode-lesshint!

**Comment by jwilsson** [comment]
I've added a link to the README. Thanks for bringing it to our attention @gnijuohz!
